### DISCUSSION TITLE
Make this an os

### DISCUSSION THREAD

**Discussion by ekahPruthvi** [prasanthrangan/hyprdots#130] [Ideas]
u k somthing like archcraft make an iso out of it
will be really cool

**Comment by prasanthrangan** [comment]
Hahaa thank you, this is still work in progress... so hopefully later!!

**Comment by MostafaMohamed2002** [comment]
i agree , it worth it

**Comment by jpx40** [comment]
I think it's enough when it's a collection of themes and scripts.
